shared/chat/conversation/messages/account-payment/container.js

```diff
@@ -77,8 +79,8 @@ const mapStateToProps = (state, ownProps: OwnProps) => {
           paymentInfo.amountDescription
         )}`,
         balanceChangeColor: WalletConstants.balanceChangeColor(paymentInfo.delta, paymentInfo.status),
-        cancelButtonInfo: youAreSender && cancelable ? makeCancelButtonInfo(theirUsername) : '',
-        cancelButtonLabel: youAreSender && cancelable ? 'Cancel' : '',
+        cancelButtonInfo: paymentInfo.showCancel ? makeCancelButtonInfo(theirUsername) : '',
+        cancelButtonLabel: paymentInfo.showCancel ? 'Cancel' : '',
```

Comment:
I'm just guessing here, but can't the cancelButtonLabel always be "Cancel" and the `showCancel` determines if it is shown or not?